src/components/media-views.js

```diff
@@ -678,5 +697,13 @@ AFRAME.registerComponent("media-image", {
     }
 
     this.el.emit("image-loaded", { src: this.data.src });
+
+    this.el.object3D.traverse(obj => {
+      obj.frustumCulled = false;
+    });
+    await AFRAME.scenes[0].systems.ticker.nextTick();
+    this.el.object3D.traverse(obj => {
+      obj.frustumCulled = true;
+    });
```

Comment:
Might be worth refactoring these lines into a single await call with a utility method, since we repeat it a few times.